SuccessfulLoginTest.txt

```diff
@@ -0,0 +1,54 @@
+package tests;
+
+
+import gui.BasePage;
+import gui.HomePage;
+import gui.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.ElementNotInteractableException;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+
+import org.testng.asserts.SoftAssert;
+
+import java.time.Duration;
+
+import static gui.BasePage.initializeDriver;
+
+public class SuccessfulLoginTest extends BasePage {
+
+    public LoginPage loginPage;
+    public HomePage homePage;
```

Comment:
There is no need to initialize HomePage here (and if it was, it's better to use private accessor)